<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>895</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ESI Term</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideLeft</enum>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Editor</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPlainTextEdit" name="plainTextEdit"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Term</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit_Term">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>895</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuESI_Term">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionClose"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionSupport"/>
   </widget>
   <widget class="QMenu" name="menuESI_Comm">
    <property name="title">
     <string>ESI Comm</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionUpload_file"/>
    <addaction name="actionComm_Mode"/>
    <addaction name="actionHome_ESI"/>
    <addaction name="actionHome_Pump"/>
   </widget>
   <addaction name="menuESI_Term"/>
   <addaction name="menuESI_Comm"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_As"/>
   <addaction name="actionClose"/>
   <addaction name="separator"/>
   <addaction name="actionSettings"/>
   <addaction name="actionUpload_file"/>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="actionComm_Mode"/>
   <addaction name="separator"/>
   <addaction name="actionHome_ESI"/>
   <addaction name="actionHome_Pump"/>
   <addaction name="actionSupport"/>
  </widget>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="esiresources.qrc">
     <normaloff>:/esi_comm/connect-serial.png</normaloff>:/esi_comm/connect-serial.png</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+C</string>
   </property>
  </action>
  <action name="actionUpload_file">
   <property name="icon">
    <iconset resource="esiresources.qrc">
     <normaloff>:/esi_comm/upload-file-icon.png</normaloff>:/esi_comm/upload-file-icon.png</iconset>
   </property>
   <property name="text">
    <string>Upload file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+U</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionSupport">
   <property name="icon">
    <iconset resource="esiresources.qrc">
     <normaloff>:/esi_comm/support-icon.png</normaloff>:/esi_comm/support-icon.png</iconset>
   </property>
   <property name="text">
    <string>Support</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+S</string>
   </property>
  </action>
  <action name="actionComm_Mode">
   <property name="icon">
    <iconset resource="esiresources.qrc">
     <normaloff>:/esi_comm/terminal-blue.png</normaloff>:/esi_comm/terminal-blue.png</iconset>
   </property>
   <property name="text">
    <string>Direct Mode</string>
   </property>
   <property name="toolTip">
    <string>Direct mode</string>
   </property>
   <property name="shortcut">
    <string>Meta+Alt+D</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="esiresources.qrc">
     <normaloff>:/esi_comm/file_open.png</normaloff>:/esi_comm/file_open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="esiresources.qrc">
     <normaloff>:/esi_comm/file_save.png</normaloff>:/esi_comm/file_save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="esiresources.qrc">
     <normaloff>:/esi_comm/BT_file_blanc_star.png</normaloff>:/esi_comm/BT_file_blanc_star.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="esiresources.qrc">
     <normaloff>:/esi_comm/file_delete.png</normaloff>:/esi_comm/file_delete.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="esiresources.qrc">
     <normaloff>:/esi_comm/file_add.png</normaloff>:/esi_comm/file_add.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="esiresources.qrc">
     <normaloff>:/esi_comm/disconnect_serial.png</normaloff>:/esi_comm/disconnect_serial.png</iconset>
   </property>
   <property name="text">
    <string>disconnect</string>
   </property>
   <property name="toolTip">
    <string>Disconnect ESI</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="esiresources.qrc">
     <normaloff>:/esi_comm/Settings-48.png</normaloff>:/esi_comm/Settings-48.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionHome_ESI">
   <property name="icon">
    <iconset resource="esiresources.qrc">
     <normaloff>:/esi_comm/blue-home-icon.png</normaloff>:/esi_comm/blue-home-icon.png</iconset>
   </property>
   <property name="text">
    <string>Home_ESI</string>
   </property>
   <property name="toolTip">
    <string>ESI origin</string>
   </property>
  </action>
  <action name="actionHome_Pump">
   <property name="icon">
    <iconset resource="esiresources.qrc">
     <normaloff>:/esi_comm/red-home-icon.png</normaloff>:/esi_comm/red-home-icon.png</iconset>
   </property>
   <property name="text">
    <string>Home_Pump</string>
   </property>
   <property name="toolTip">
    <string>Pump origin</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="esiresources.qrc"/>
 </resources>
 <connections/>
</ui>
